src/components/diagrams/singleLineDiagram/single-line-diagram-pane.js

```diff
@@ -205,7 +205,7 @@ export function SingleLineDiagramPane({
                 ?.ref?.current?.reloadSvg();
         } else
             viewsRef.current.forEach((sld) => {
-                if (sld.svgUrl.indexOf(currentNodeRef.current?.id) !== -1) {
+                if (sld.svgUrl?.indexOf(currentNodeRef.current?.id) !== -1) {
```

Comment:
Hum, dangerous. if `sld.svgUrl` is undefined, the test `!== -1` will succeed, and I guess this is not what you want.

Reply:
that is right. fixed it